arm/Microsoft.Storage/storageAccounts/readme.md

```diff
@@ -1,6 +1,6 @@
-# StorageAccounts `[Microsoft.Storage/storageAccounts]`
+# Storage Accounts `[Microsoft.Storage/storageAccounts]`
 
-This module is used to deploy an Azure Storage Account, with resource lock and the ability to deploy 1 or more Blob Containers and 1 or more File Shares. Optional ACLs can be configured on the Storage Account and optional RBAC can be assigned on the Storage Account and on each Blob Container and File Share.
+This module is used to deploy a storage account, with resource lock and the ability to deploy 1 or more blob containers, file shares, tables and queues. Optional ACLs can be configured on the storage account and optional RBAC can be assigned on the storage account and on each child resource.
 
 The default parameter values are based on the needs of deploying a diagnostic storage account.
```

Comment:
Is the sentence with the diagnostic storage account still true? Not sure what a storage account should be capable of to be called 'diagnostic storage account'

Reply:
done